plugins/lookup/dependent.py

```diff
@@ -144,7 +144,7 @@ def __evaluate(self, expression, templar, variables):
         ``variables`` are the variables to use.
         """
         templar.available_variables = variables or {}
-        expression = "{0}{1}{2}".format("{{", expression, "}}")
+        expression = f"{{{{{expression}}}}}"
```

Comment:
The idea was to have `{{` and `}}` explicitly as strings here, in case these are made configurable at some point. How about:
```suggestion
        expression = f"{'{{'}{expression}{'}}'}"
```

Reply:
I got one that is the same length but arguably slightly better to read:
```python
        expression = ('{{' f'{expression}' '}}')
```

Reply:
Same here, maybe not using f-strings in this case would be better?

Reply:
you're right, this case is simpler without it